internal/pkg/manifest/workload.go

```diff
@@ -408,15 +407,16 @@ func (lc *Logging) GetEnableMetadata() *string {
 
 // SidecarConfig represents the configurable options for setting up a sidecar container.
 type SidecarConfig struct {
-	Port          *string             `yaml:"port"`
-	Image         *string             `yaml:"image"`
-	Essential     *bool               `yaml:"essential"`
-	CredsParam    *string             `yaml:"credentialsParameter"`
-	Variables     map[string]string   `yaml:"variables"`
-	Secrets       map[string]string   `yaml:"secrets"`
-	MountPoints   []SidecarMountPoint `yaml:"mount_points"`
-	DockerLabels  map[string]string   `yaml:"labels"`
-	DependsOn     map[string]string   `yaml:"depends_on"`
+	Port          *string              `yaml:"port"`
+	Image         *string              `yaml:"image"`
+	Essential     *bool                `yaml:"essential"`
+	CredsParam    *string              `yaml:"credentialsParameter"`
+	Variables     map[string]string    `yaml:"variables"`
+	Secrets       map[string]string    `yaml:"secrets"`
+	MountPoints   []SidecarMountPoint  `yaml:"mount_points"`
+	DockerLabels  map[string]string    `yaml:"labels"`
+	DependsOn     map[string]string    `yaml:"depends_on"`
+	HealthCheck   ContainerHealthCheck `yaml:"healthcheck"`
```

Comment:
Should we add this now into:
https://github.com/aws/copilot-cli/blob/ecc3cf92643630cc1cf741cb5bf05b3e1e335472/internal/pkg/manifest/validate.go#L520-L527